src/main/java/com/google/jenkins/plugins/k8sengine/ClusterUtil.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.jenkins.plugins.k8sengine;
+
+import com.google.api.services.container.model.Cluster;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+
+/** Utility functions for converting between {@link Cluster}s and their String representations. */
+class ClusterUtil {
+
+  /**
+   * Given a GKE {@link Cluster} return a String representation containing the name and zone.
+   * @param cluster The non-null {@link Cluster} object.
+   * @return A String of the form "name (zone)" based on the given cluster's properties.
+   */
+  static String toNameAndZone(Cluster cluster) {
+    Preconditions.checkNotNull(cluster);
+    return toNameAndZone(cluster.getName(), cluster.getZone());
+  }
+
+  /**
+   * Given a name and zone for a cluster, return the combined String representation.
+   * @param name A non-null, non-empty cluster name
```

Comment:
No need to specify non-null in the description. Any non-optional param is assumed to be non-null.

Reply:
Done.